intern/service/boss/v1/volume.py

```diff
@@ -251,7 +251,13 @@ def get_cutout(
             ), dtype=resource.datatype)
 
             if parallel:
-                pool = multiprocessing.Pool(processes=parallel if isinstance(parallel, int) and parallel > 0 else multiprocessing.cpu_count())
+                if type(parallel) == bool:
```

Comment:
Let's remove the type annotation from the function or make it `Union[bool, int]` so it's not a lie. Also update the docstring to indicate that an int is also a valid type.

Reply:
Still need to also hit the type annotation in the function def, itself.

Reply:
Ok removed the type annotation in the v1 volume.py. By the function def did you mean the wrapper function in `service/volume.py`?

Reply:
I was referring to the change you just made.